File: MiniMaxSum.cs
using System.CodeDom.Compiler;
using System.Collections.Generic;
using System.Collections;
using System.ComponentModel;
using System.Diagnostics.CodeAnalysis;
using System.Globalization;
using System.IO;
using System.Linq;
using System.Reflection;
using System.Runtime.Serialization;
using System.Text.RegularExpressions;
using System.Text;
using System;

class Solution {

    // Complete the miniMaxSum function below.
    static void miniMaxSum(int[] arr) {
        UInt64 max = 1;
        UInt64 min = 4000000000;
        for (var i = 0; i < arr.Length; i++)
        {
            UInt64 sum = 0;
            for (var j = 0; j < arr.Length; j++)
            {
                if (i != j)
                {
                    sum += (UInt64)arr[j];
                }
            }
            max = Math.Max(sum,max);
            min = Math.Min(sum,min);
        }
        Console.WriteLine(min + " " + max);
    }

    static void Main(string[] args) {
        int[] arr = Array.ConvertAll(Console.ReadLine().Split(' '), arrTemp => Convert.ToInt32(arrTemp))
        ;
        miniMaxSum(arr);
    }
}

File: MiniMaxSum.cpp
#include <bits/stdc++.h>
#include <algorithm>

using namespace std;

int main() {
    vector<int> arr(5);
    long long int s1=0,s2=0,s3=0,s4=0,s5 = 0;
    for(int arr_i = 0; arr_i < 5; arr_i++){
       cin >> arr[arr_i];
        if (arr_i != 0) s1 += arr[arr_i];
        if (arr_i != 1) s2 += arr[arr_i];
        if (arr_i != 2) s3 += arr[arr_i];
        if (arr_i != 3) s4 += arr[arr_i];
        if (arr_i != 4) s5 += arr[arr_i];
    }
    long long int minimum = min(min(min(min(s1, s2), s3), s4), s5);
    long long int maximum = max(max(max(max(s1, s2), s3), s4), s5);

    cout << minimum << " " << maximum;

    return 0;
}

